src/main/java/org/jabref/gui/sidepane/GroupsSidePaneHeaderView.java

```diff
@@ -0,0 +1,56 @@
+package org.jabref.gui.sidepane;
+
+import javafx.scene.control.Button;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.actions.SimpleCommand;
+import org.jabref.gui.groups.GroupModeViewModel;
+import org.jabref.gui.groups.GroupViewMode;
+import org.jabref.gui.icon.IconTheme;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.preferences.PreferencesService;
+
+public class GroupsSidePaneHeaderView extends SidePaneHeaderView {
```

Comment:
I guess because I'm used to FXML and it felt that the header view construction logic should have its own class/file, but mostly because I'm overengineering :), I implemented your suggestion by merging the header view into sidepaneview